fsthttp/limits.go

```diff
@@ -0,0 +1,49 @@
+package fsthttp
+
+import "github.com/fastly/compute-sdk-go/internal/abi/fastly"
+
+// Limits handles HTTP limits
+var Limits = limits{}
+
+type limits struct {
+}
```

Comment:
I'd put the limits in the struct, that's four fewer globals...

Reply:
It would create a circular dependency between the `fsthttp` and `internal/abi/fastly` packages.

Reply:
emoji-homer-hedge :-)